src/lib/log/audit/audit.go

```diff
@@ -0,0 +1,148 @@
+//  Copyright Project Harbor Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package audit
+
+import (
+	"context"
+	"fmt"
+	"log/syslog"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/goharbor/harbor/src/lib/cache"
+	"github.com/goharbor/harbor/src/lib/config"
+	"github.com/goharbor/harbor/src/lib/log"
+)
+
+const (
+	endpointHealthCheckInterval = 1 * time.Minute
+	cacheInterval               = 2 * time.Minute
+)
+
+var LogMgr = &LoggerManager{}
```

Comment:
why do we put the audit log manager into lib?

Reply:
move to pkg/audit